platforms/ubuntutouch/gcompris.desktop

```diff
@@ -88,7 +88,7 @@ GenericName[ta]=குழந்தைகளுக்கான கல்விச
 GenericName[tg]=Бозиҳои омӯзишӣ барои кӯдакон
 GenericName[tr]=Çocuklar için Eğitici Oyunlar
 GenericName[uk]=Освітня гра для дітей
-GenericName[x-test]=xxEducational Game for Childrenxx
+#GenericName[x-test]=xxEducational Game for Childrenxx
```

Comment:
we won't be able to commit this change as this part of the file is filled automatically by KDE scripts and we can't ignore it.

Reply:
that desktop file is a copy of the root dir one. Is it still an issue ?. 
I can rollback it if needed. This is just that our build system doesn't support x-test name for whatever reason

Reply:
yes it fails when building.

Reply:
According to the KDE i18n team:
```
We don't
but we should really remove the x-test translations at this point
there's just no way to use them at this point
```

I'll take a look if I can help them to remove these ones